nix/shared.nix

```diff
@@ -393,6 +393,10 @@ let
                           haskellPackagesNew.semigroups
                         ];
 
+                    hnix =
+                      pkgsNew.haskell.lib.doJailbreak
+                        haskellPackagesOld.hnix;
```

Comment:
So I thought that this would allow `hnix` to be built with `prettyprinter-1.5.1`, but [Hydra seems to disagree](https://hydra.dhall-lang.org/build/44511/nixlog/1):

```
Configuring hnix-0.6.1...
CallStack (from HasCallStack):
  die', called at libraries/Cabal/Cabal/Distribution/Simple/Configure.hs:958:20 in Cabal-2.2.0.1:Distribution.Simple.Configure
  configureFinalizedPackage, called at libraries/Cabal/Cabal/Distribution/Simple/Configure.hs:462:12 in Cabal-2.2.0.1:Distribution.Simple.Configure
  configure, called at libraries/Cabal/Cabal/Distribution/Simple.hs:596:20 in Cabal-2.2.0.1:Distribution.Simple
  confHook, called at libraries/Cabal/Cabal/Distribution/Simple/UserHooks.hs:67:5 in Cabal-2.2.0.1:Distribution.Simple.UserHooks
  configureAction, called at libraries/Cabal/Cabal/Distribution/Simple.hs:178:19 in Cabal-2.2.0.1:Distribution.Simple
  defaultMainHelper, called at libraries/Cabal/Cabal/Distribution/Simple.hs:115:27 in Cabal-2.2.0.1:Distribution.Simple
  defaultMain, called at Setup.hs:2:8 in main:Main
Setup: Encountered missing dependencies:
prettyprinter >=1.2.1 && <1.3
```

Could you take a look @Gabriel439?

Reply:
@sjakobi: I pushed a change to fix the Nix build

Reply:
Thank you @Gabriel439! :)

Reply:
@sjakobi: You're welcome! 🙂